src/test/java/com/ss/ita/google/SearchAndReturnHomeTest.java

```diff
@@ -18,4 +19,17 @@ public void verifyReturningHomePageAfterSearching() {
                 .returnHomePage();
         assertTrue(driver.findElement(By.id("hplogo")).isDisplayed());
     }
+
+    @Test
+    public void verifySearchLinksContainsWord() {
+        SearchResultPage searchResultPage = new HomePage()
+                .doSearch("funny kitten");
+
+        String word = "kitten";
+        assertTrue(searchResultPage.getResultLinksTexts().size() != 0);
+        for (int i = 0; i < searchResultPage.getResultLinksTexts().size(); i++) {
+            String linkText = searchResultPage.getLinkText(i).toLowerCase();
+            assertTrue(linkText.contains(word));
```

Comment:
searchResultPage.getResultLinksTexts().size() - extract to a variable
use for each